use_cases/eluc/prescriptors/nsga2/torch_prescriptor.py

```diff
@@ -145,9 +145,7 @@ def _select_parents(self, candidates: list[Candidate], n_parents: int) -> list[C
         for front in fronts:
             # Compute crowding distance here even though it's technically not necessary now
             # so that later we can sort by distance
-            crowding_distance = nsga2_utils.calculate_crowding_distance(front)
-            for candidate, distance in zip(front, crowding_distance):
-                candidate.distance = distance
+            nsga2_utils.calculate_crowding_distance(front)
```

Comment:
Change from manually setting distances outside function to changing them inplace